central/policy/service/service_impl.go

```diff
@@ -727,6 +727,11 @@ func (s *serviceImpl) ExportPolicies(ctx context.Context, request *v1.ExportPoli
 		return nil, err
 	}
 	if len(policyErrors) > 0 {
+		for i, missingIndex := range missingIndices {
+			policyError := policyErrors[i].Error()
+			policyID := request.PolicyIds[missingIndex]
+			log.Warnf("A policy error ocurred for id %s: '%v'", policyID, policyError)
```

Comment:
we are already doing this loop in line 736. Can you add the log statement there?